packages/azure-services/src/storage/cosmos-container-client.ts

```diff
@@ -14,12 +14,17 @@ import { RetryOptions } from './retry-options';
 
 // tslint:disable: no-any no-unsafe-any
 
+async function defaultSleepFunction(millis: number): Promise<void> {
+    await System.wait(millis);
+}
+
```

Comment:
seems we have redundant func declaration
will `private readonly sleepFunction: (ms: number) => Promise<void> = System.wait` work?